### PR TITLE
Listing all meetup locations and map.

### PR DESCRIPTION
**Pull request by exploreshaifali**
I created a separate meetup_locations_map.html snippet for map and included in list_locations.html template even though snippet contain most of js code because I could not find any other optimal option to use Django template tags to use with js.

### CODE REVIEW

File: systers_portal/templates/meetup/list_location.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block title %}
+  Systers Meetup Locations
+{% endblock %}
+
+{% block content %}
+  <h2 class="mt40 text-center">Systers Meetup Locations</h2>
+  <div class="row">
+    <div class="col-xs-offset-1 col-xs-10 col-xs-offset-1 mt20">
```
Comment (ana-balica) [inline systers_portal/templates/meetup/list_location.html]: Just a suggestion: what if we leave the map width to take the whole available space and increase the height a bit, say 400px.

File: systers_portal/templates/meetup/snippets/meetup_locations_grid.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<div class="row box-body">
+  <div class="col-xs-offset-1 col-xs-5">
```
Comment (ana-balica) [inline systers_portal/templates/meetup/snippets/meetup_locations_grid.html]: And for the rows we can do the following: remove the offset, just use the columns. Set a half for a each div with `col-md-6` for medium devices and for small ones take the full width using `col-xs-12`. So on big screens we will get 2 columns, but on small ones just 1. 

Everything else looks very good :+1:

File: systers_portal/templates/meetup/list_location.html (hunk 1 above)
Comment (exploreshaifali) [inline systers_portal/templates/meetup/list_location.html]: Okay.

File: systers_portal/templates/meetup/snippets/meetup_locations_grid.html (hunk 2 above)
Comment (exploreshaifali) [inline systers_portal/templates/meetup/snippets/meetup_locations_grid.html]: Cool :)

**Pull request merged**